cg/constants/housekeeper_tags.py

```diff
@@ -1,11 +1,40 @@
 """File tags for files in Housekeeper."""
-
+from enum import Enum
 from typing import List
 
 from cgmodels.cg.constants import Pipeline, StrEnum
 
 
+class AlignmentFileTag(str, Enum):
```

Comment:
For consistency in this file maybe inherit from StrEnum as we import it?
```suggestion
class AlignmentFileTag(StrEnum):
```